spot-client/src/common/remote-control/BaseRemoteControlService.js

```diff
@@ -87,6 +87,9 @@ export class BaseRemoteControlService extends Emitter {
             onPresenceReceived: this._onPresenceReceived
         });
 
+        !this._p2pSignaling || logger.error('Leaked P2PSignaling instance');
```

Comment:
Do you not want to actively prevent the leak?

Reply:
It is preventing the leak by doing the cleanup. It is just a debug log message to catch the corner case and save time on debugging in case the cleanup is broken.

Reply:
Oh you mean to do the cleanup when it happens, no because someone could think it is a supported case.